kafka/src/main/java/no/nav/common/kafka/consumer/KafkaConsumerClientImpl.java

```diff
@@ -148,6 +145,15 @@ private void consumeTopics() {
 
                 processedRecordsLatch = new CountDownLatch(totalRecords);
 
+                /*
+                 We seek to the first offset of each topic+partition so that all the records in the batch must
+                 be processed before polling new records.
+
+                 If this is not done then it is possible for records from revoked partitions or failed records to be skipped if
+                 no other record for that topic+partition has been added to offsetsToCommit.
+                */
+                seekToFirstOffsetForEachTopicPartition(records);
```

Comment:
Fra doc til seek: `* @throws IllegalStateException if the provided TopicPartition is not assigned to this consumer`. Kan dette skje her? Bør vi ha `try catch`?

Reply:
Trengs ikke